ivy/functional/backends/jax/activations.py

```diff
@@ -45,36 +45,28 @@ def sigmoid(x: JaxArray, /, *, out: Optional[JaxArray] = None) -> JaxArray:
 
 
 def softmax(
-    x: JaxArray, /, *, axis: Optional[int] = None, out: Optional[JaxArray] = None
+    x: JaxArray, /, *, axis: int = -1, out: Optional[JaxArray] = None
 ) -> JaxArray:
-    if axis is None:
-        axis = -1
     return jax.nn.softmax(x, axis)
```

Comment:
But first I would suggest fixing the jax backend of `ivy.softmax`. I can see `test_softmax` is also failing there and this might be related to some frontend failures.